content/pt-br/docs/reference/glossary/shuffle-sharding.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Shuffle-sharding
+id: shuffle-sharding
+date: 2020-03-04
+full_link:
+short_description: >
+  Uma técnica para atribuir requisições para filas que proporciona melhor isolamento do que hashing módulo do número de filas.
+
+aka:
+tags:
+- fundamental
+---
+Uma técnica para atribuir requisições para filas que proporciona melhor isolamento do que hashing módulo do número de filas.
+
+<!--more-->
+
+Nos preocupamos frequentemente em isolar diferentes fluxos de requisições
+uma das outras, de modo que um fluxo de alta intensidade não afete um fluxo de baixa intensidade.
+Uma forma simples de colocar requisições em fila é gerar um hash baseado
+em características da requisição, módulo do número de filas, para ter o
+índice da fila à ser usada. A função hash usa como entrada características da requisição
+que se alinha com o fluxo. Por exemplo, na Internet isso é frequentemente usada as 5 tuplas de
+endereço de origem e destino, protocolo, e portas de origem e destino.
```

Comment:
```suggestion
que se alinha com o fluxo. Por exemplo, na Internet isso é frequentemente a tupla de 5 elementos de
endereço de origem e destino, protocolo, e portas de origem e destino.
```

"5-tuple" here stands for the [five different elements that build said tuple](https://datatracker.ietf.org/doc/html/rfc6146#autoid-7), not for 5 different tuples.